File: Leetcode/Array/#485. Max Consecutive Ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int res=INT_MIN;
        int cnt=0;
        for(int i=0; i<nums.size(); i++){
            if(nums[i]==1) res = max(++cnt,res);
            else cnt=0;
        }
        return res==INT_MIN ? 0 : res; 
    }
};

File: Array/#485. Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        nums.append(0)
        cnt = 0
        res = 0

        for e in nums:
            if e == 0:
                res = max(res, cnt)
                cnt = 0
            else:
                cnt += 1

        return res